kubernetes/is-secrets.yaml

```diff
@@ -3,7 +3,7 @@ kind: Secret
 metadata:
   name: integration-service-api-secrets
 type: Opaque
-data:
-  SERVER_SECRET: UHBLRmhQS0pZMmVmVHNOOVZrQjdXTnRZVWhYOVV0YWE=
-  API_KEY: OTRGNUJBNDktMTJCNi00RTQ1LUE0ODctQkY5MUM0NDIyNzZE
+stringData:
```

Comment:
why is this now stringData?

Reply:
Is only the new way to create secret avoiding to write them as Base64. The outcome is the same just simplify the life to DevOps

Reply:
thanks